File: app/src/main/java/ru/argerd/repo/ui/detailDescriptionScreen/Decorator.java
package ru.argerd.repo.ui.detailDescriptionScreen;

import android.graphics.Rect;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

class Decorator extends RecyclerView.ItemDecoration {
    private float marginFirst;
    private float marginTopBottom;
    private float margin;

    Decorator(float marginStart, float marginTopBottom, float margin) {
        this.marginFirst = marginStart;
        this.marginTopBottom = marginTopBottom;
        this.margin = margin;
    }

    @Override
    public void getItemOffsets(@NonNull Rect outRect, @NonNull View view,
                               @NonNull RecyclerView parent, @NonNull RecyclerView.State state) {
        outRect.top = (int) marginTopBottom;
        outRect.bottom = (int) marginTopBottom;
        if (parent.getChildLayoutPosition(view) == 0) outRect.left = (int) marginFirst;
        else if (parent.getChildLayoutPosition(view) != state.getItemCount() - 1) {
            outRect.left = (int) margin;
        }
    }
}

File: app/src/main/java/ru/argerd/repo/ui/newsScreen/Decorator.kt
package ru.argerd.repo.ui.newsScreen

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView

internal class Decorator(
        private val margin: Int
) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View,
                                parent: RecyclerView, state: RecyclerView.State) {
        if (parent.getChildLayoutPosition(view) == state.itemCount - 1) {
            outRect.bottom = margin
        }
    }
}